conf/application.conf

```diff
@@ -197,4 +197,7 @@ security {
 
 message-translation-file = "MessageTranslation.json"
 
-jdk.xml.maxOccurLimit = 10000
\ No newline at end of file
+jdk.xml.maxOccurLimit = 10000
+
+defaultGuarantee.currency = GBP
+defaultGuarantee.amount = 10.00
```

Comment:
£10 or £1 ?

Reply:
🤦 I left that in to confirm it was working. let me revert that